lmdeploy/pytorch/kernels/pagedattention.py

```diff
@@ -413,6 +569,8 @@ def paged_attention_fwd(
     kv_seqlens: Tensor,
     max_seqlen: int,
     window_size: int = None,
+    sm_scale: float = None,
+    shared_kv: int = False,
```

Comment:
int -> bool. 
what does `shared_kv` mean?

Reply:
https://kexue.fm/archives/10091#Part%203

K is `[ci, rope]` V is `[ci]`. V share the same memory with K.

And it is not recommend to enable this flag since the layout of shared V is not friendly to matmul.